dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -34,14 +34,37 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 				Rect(0, 0, 0, 0)
 			} else boundings[0]
 		}
+	private val isStatusBarVisible: Boolean
+		get() {
+			val rectangle = Rect()
+			val window: Window? = currentActivity?.window;
+			window?.decorView?.getWindowVisibleDisplayFrame(rectangle)
+			val statusBarHeight = rectangle.top
+			return statusBarHeight != 0
+		}
+	private val mStatusBarHeight: Int
+		get() {
+			var statusBarHeight: Int = 0;
+			if(isStatusBarVisible)
+			{
+				val resourceId: Int = reactApplicationContext.resources.getIdentifier("status_bar_height", "dimen", "android")
+				if (resourceId > 0) {
+					statusBarHeight = reactApplicationContext.resources.getDimensionPixelSize(resourceId)
+				}
+				return statusBarHeight;
+			}
+			return statusBarHeight;
+		}
 	private val windowRects: List<Rect>
 		get() {
 			val boundings = mDisplayMask?.getBoundingRectsForRotation(rotation)
 			val windowBounds = windowRect;
-			return if (boundings == null  || boundings.size == 0) {
+			return if (boundings == null || boundings.size == 0) {
+				windowBounds.bottom = windowBounds.bottom - mStatusBarHeight;
 				listOf(windowBounds)
 			} else {
 				val hingeRect = boundings[0]
+				hingeRect.bottom = hingeRect.bottom - mStatusBarHeight;
```

Comment:
>hingeRect.bottom = hingeRect.bottom - mStatusBarHeight; [](start = 4, length = 55)

This line of code should move to inside the else block (for clarity as it's only necessary for that case), and you should also subtract the status bar height from hingeRect.top, otherwise the top rect will remain too tall and go under the hinge.